requirements.txt

```diff
@@ -6,7 +6,7 @@ pbr<5,>=4.0 # Apache-2.0
 aiohttp<4,>=3.2.1
 appdirs<2,>=1.4.3
 webargs<6,>=5
-marshmallow<4,>=3.0.0rc4
+marshmallow<3.0.0rc7,>=3.0.0rc4
```

Comment:
I think rc5 is the latest working version

    marshmallow<3.0.0rc6,>=3.0.0rc4